pettingzoo/test/api_test.py

```diff
@@ -62,39 +58,17 @@ def test_observation_action_spaces(env, agent_0):
         assert isinstance(
             env.action_spaces[agent], gym.spaces.Space
         ), "Agent space for each agent must extend gym.spaces.Space"
-        if not (
-            isinstance(env.observation_spaces[agent], gym.spaces.Box)
-            or isinstance(env.observation_spaces[agent], gym.spaces.Discrete)
-        ):
-            warnings.warn(
-                "Observation space for each agent probably should be gym.spaces.box or gym.spaces.discrete"
-            )
-        if not (
-            isinstance(env.action_spaces[agent], gym.spaces.Box)
-            or isinstance(env.action_spaces[agent], gym.spaces.Discrete)
-        ):
-            warnings.warn(
-                "Action space for each agent probably should be gym.spaces.box or gym.spaces.discrete"
-            )
+        if not (isinstance(env.observation_spaces[agent], (gym.spaces.Box, gym.spaces.Discrete))):
+            warnings.warn("Observation space for each agent probably should be gym.spaces.box or gym.spaces.discrete")
+        if not (isinstance(env.action_spaces[agent], (gym.spaces.Box, gym.spaces.Discrete))):
+            warnings.warn("Action space for each agent probably should be gym.spaces.box or gym.spaces.discrete")
```

Comment:
The second parameter of `isinstance()` can be a tuple().
* https://docs.python.org/3/library/functions.html?highlight=isinstance#isinstance